package.json

```diff
@@ -10,6 +10,7 @@
     "preinstall": "./install-hooks",
     "test": "grunt test",
     "test:karma": "karma start tests/karma/all.conf.js",
+    "test:karma-single-test": "karma start tests/karma/single.conf.js",
```

Comment:
please add info in README also about running single test

Reply:
Done